python/semantic_kernel/contents/streaming_text_content.py

```diff
@@ -1,4 +1,10 @@
 # Copyright (c) Microsoft. All rights reserved.
+import sys
+
+if sys.version < "3.11":
+    pass
+else:
+    pass
```

Comment:
:) what? :D :D

Reply:
Deleting the Self from the code, creates this, will remove